drivers/storage/portworx/component/securitycontextconstraints.go

```diff
@@ -99,12 +105,20 @@ func (s *scc) Reconcile(cluster *opcorev1.StorageCluster) error {
 			}
 		} else if err == nil {
 			resourceVersion := existingScc.ResourceVersion
-			pxutil.CleanupObject(&scc)
-			pxutil.CleanupObject(&existingScc)
-			logrus.Infof("Updating %s security context constraints", scc.Name)
-			if !reflect.DeepEqual(scc, existingScc) {
+			sccPriority := *scc.Priority
+			existingSccPriority := *existingScc.Priority
+			sccAllowPrivilegeEscalation := *scc.AllowPrivilegeEscalation
+			existingSccAllowPrivilegeEscalation := *existingScc.AllowPrivilegeEscalation
+			cleanupSccObject(&scc)
+			cleanupSccObject(&existingScc)
+			if sccPriority != existingSccPriority ||
+				sccAllowPrivilegeEscalation != existingSccAllowPrivilegeEscalation ||
+				!equality.Semantic.DeepDerivative(scc, existingScc) {
 				scc.ResourceVersion = resourceVersion
+				scc.Priority = &sccPriority
 				err = s.k8sClient.Update(context.TODO(), &scc)
+				scc.AllowPrivilegeEscalation = &sccAllowPrivilegeEscalation
+				logrus.Infof("Updating %s security context constraints", scc.Name)
```

Comment:
move this to one line before the k8sClient.Update API call

Reply:
Will update